prime-router/src/test/kotlin/azure/SendFunctionTests.kt

```diff
@@ -97,14 +98,15 @@ class SendFunctionTests {
     @Test
     fun `Test with message`() {
         // Setup
-        var nextEvent: ReportEvent? = null
+        var nextEvent1: ReportEvent? = null
```

Comment:
good experiment, put these back to `nextEvent`, the numbering will make people look for other related variables within the same scope.

Reply:
done.